### DISCUSSION TITLE
Search in bookmarks

### DISCUSSION THREAD

**Discussion by boubkhaled** [sumatrapdfreader/sumatrapdf#4241] [Ideas]
## Is there a way to search in bookmarks?
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/18572114/aded60f9-3e30-4104-ac17-6649330b192d)

**Comment by GitHubRulesOK** [comment]
There is a very limited ability to search in that Windows provided pane. It is marginally better than native Acrobat ability with Bookmarks

It is SAME NAVIGATOR FUNCTION as provided by Microsoft Explorer
It works at currently expanded entries and is "DownWards"

That Entry is within current view CoverPage +P+P (OR from closed state F + Right + O + Right + P)
